### PR TITLE
JSX Tag Name is not same | Snipped works now

### PR DESCRIPTION
**Pull request by BenjaminZekavica**
Change correct Tag Name.

### CODE REVIEW

**Review by Soean — APPROVED** [review]
Thanks, good catch!

File: packages/edit-post/README.md [hunk 1]
```diff
@@ -134,9 +134,9 @@ const { registerPlugin } = wp.plugins;
 const { PluginDocumentSettingPanel } = wp.editPost;
 
 const MyDocumentSettingTest = () => (
-		<PluginDocumentSettingPanel className="my-document-setting-plugin">
-		<p>My Document Setting Panel</p>
-	</PluginDocumentSetting>
+	<PluginDocumentSettingPanel className="my-document-setting-plugin">
+	    <p>My Document Setting Panel</p>
```
Comment (Soean) [inline packages/edit-post/README.md]:
```suggestion
		<p>My Document Setting Panel</p>
```
Minor: Indentation with tabs, not spaces

**Review by Soean — CHANGES_REQUESTED** [review]
Oh, I was too fast. The tests are failing, because this file is generated from the source files. You need to change the source in the file `/packages/edit-post/src/components/sidebar/plugin-document-setting-panel/index.js` in line 73 and then run the command `npm run docs:build` to generate the `Readme.md` file.

**Comment by BenjaminZekavica** [comment]
@Soean Now it's build :)

**Review by Soean — APPROVED** [review]
Tests are green, now its working, thanks!

**Pull request merged**